src/ci.yml

```diff
@@ -62,13 +62,32 @@ jobs:
       {
         throw "UseCoreToolsBuildFromIntegrationTests can only be set to True or False. Current value is set to $env:USECORETOOLSBUILDFROMINTEGRATIONTESTS"
       }
-      ./setup-e2e-tests.ps1 -FunctionsRuntimeVersion $env:FUNCTIONSRUNTIMEVERSION `
+      ./setup-e2e-tests.ps1 -DotnetVersion 'net7' `
                             -UseCoreToolsBuildFromIntegrationTests:$useIntegrationTestingCoreTools `
                             -SkipBuildOnPack
-    displayName: "Setup E2E tests"
+    displayName: "Setup E2E tests - Net7"
 
   - task: DotNetCoreCLI@2
-    displayName: 'Run tests'
+    displayName: 'Run tests - Net7'
+    inputs:
+      command: 'test'
+      arguments: '--no-build -c Release'
+      projects: |
+        test/**/*Tests.csproj
+
+  - pwsh: |
+      $useIntegrationTestingCoreTools = $null
+      if (-not ([bool]::TryParse($env:USECORETOOLSBUILDFROMINTEGRATIONTESTS, [ref] $useIntegrationTestingCoreTools)))
+      {
+        throw "UseCoreToolsBuildFromIntegrationTests can only be set to True or False. Current value is set to $env:USECORETOOLSBUILDFROMINTEGRATIONTESTS"
+      }
+      ./setup-e2e-tests.ps1 -DotnetVersion 'netfx' `
+                            -UseCoreToolsBuildFromIntegrationTests:$useIntegrationTestingCoreTools `
+                            -SkipBuildOnPack
+    displayName: "Setup E2E tests - NetFramework"
+
+  - task: DotNetCoreCLI@2
+    displayName: 'Run tests - NetFramework'
```

Comment:
just to confirm, we are not running these for linux right?

Reply:
Skipped Netframework tests for linux